news/2025/2025-02-13-osumania-7k-world-cup-2025-semifinals-recap.md

```diff
@@ -0,0 +1,285 @@
+
+You may've noticed by now the trend, with both teams winning points by rather narrow margins and switching their leads back and forth. To be expected from a match with two closely matches teams! This trend continued till the very end of the match, with China ultimately gaining an advantage over the Philippines when they achieved their second break point by stealing [Elliot Hsu - A Guiding Star](https://osu.ppy.sh/beatmapsets/2317897#mania/4964065) (LN1) from their rivals' hands. A very important point to win, as the 2-point lead would be dragged throughout the whole match, all thanks to ::{ flag=CN }:: [[Crz]Satori](https://osu.ppy.sh/users/7082178)'s crazy 987,943 full combo!
+
+With a 3-point deficit over their rivals after losing [Lime - Pixel Planet](https://osu.ppy.sh/beatmapsets/2317895#mania/4964061) (RC6), the Philippines begun their comeback by picking [mitei - Phosphor](https://osu.ppy.sh/beatmapsets/2317893#mania/4964059) (HB3). With ::{ flag=PH }:: [Ojisan-](https://osu.ppy.sh/users/11420405) still going strong despite the pick's difficulty, and their rivals' lack of consistency on the pick, the Philippines managed to gain some ground back. Having their backs against the wall, China had no choice but to pick [Flying Steps - We Are Electric](https://osu.ppy.sh/beatmapsets/2317884#mania/4964044) (RC3) which was not their best of options. Despite their best efforts however, the Philippines gained the upper hand on the map thanks to ::{ flag=PH }:: [Ojisan-](https://osu.ppy.sh/users/11420405) and ::{ flag=PH }:: [sanni666](https://osu.ppy.sh/users/3036686)'s performance with quite a small 5,843-point lead, positioning themselves on a very comfortable spot to dispute China's match point.
+
+The last gambit from the Filipinos was with their very last pick of the match, being [Xyris - FOXGLOVE](https://osu.ppy.sh/beatmapsets/2318030#mania/4964416) (HB1). A point that was heavily contested by both teams, but that ultimately was won by the Chinese thanks to their first-class consistency! A victory well coveted by team China, who managed to win the match with a 6–4 score and secure themselves a spot on the podium.
+
+### ::{ flag=HK }:: Hong Kong vs. ::{ flag=PE }:: Peru
```

Comment:
this match isn't really that spectacular compared to the others, what's the reasoning for the coverage?

Reply:
This should've been discussed right after the weekend's matches when matches get decided...

Peru's performance both throughout the match and tournament were fitting for it to be covered. Considering they were going to be interviewed, it seems like a good prelude to it.

Reply:
seems like valid reasoning
also the section itself is well written anyway, i'd say